meson.build

```diff
@@ -0,0 +1,308 @@
+    'apps/blueman-sendto',
+    'apps/blueman-services',
+    'apps/blueman-tray',
+    install_dir: bindir,
+    install_mode: 'rwxr-xr-x'
+)
+
+install_data(
+    'apps/blueman-mechanism',
+    'apps/blueman-rfcomm-watcher',
+    install_dir: libexecdir,
+    install_mode: 'rwxr-xr-x'
+)
+
+
+# Install all the artwork and icons
+# FIXME use strip_directory when meson 0.45.0 is mainstream
+install_data(
+    'data/icons/pixmaps/blueman-lq-10.png',
+	'data/icons/pixmaps/blueman-lq-20.png',
+	'data/icons/pixmaps/blueman-lq-30.png',
+	'data/icons/pixmaps/blueman-lq-40.png',
+	'data/icons/pixmaps/blueman-lq-50.png',
+	'data/icons/pixmaps/blueman-lq-60.png',
+	'data/icons/pixmaps/blueman-lq-70.png',
+	'data/icons/pixmaps/blueman-lq-80.png',
+	'data/icons/pixmaps/blueman-lq-90.png',
+	'data/icons/pixmaps/blueman-lq-100.png',
+	'data/icons/pixmaps/blueman-rssi-10.png',
+	'data/icons/pixmaps/blueman-rssi-20.png',
+	'data/icons/pixmaps/blueman-rssi-30.png',
+	'data/icons/pixmaps/blueman-rssi-40.png',
+	'data/icons/pixmaps/blueman-rssi-50.png',
+	'data/icons/pixmaps/blueman-rssi-60.png',
+	'data/icons/pixmaps/blueman-rssi-70.png',
+	'data/icons/pixmaps/blueman-rssi-80.png',
+	'data/icons/pixmaps/blueman-rssi-90.png',
+	'data/icons/pixmaps/blueman-rssi-100.png',
+	'data/icons/pixmaps/blueman-tpl-10.png',
+	'data/icons/pixmaps/blueman-tpl-20.png',
+	'data/icons/pixmaps/blueman-tpl-30.png',
+	'data/icons/pixmaps/blueman-tpl-40.png',
+	'data/icons/pixmaps/blueman-tpl-50.png',
+	'data/icons/pixmaps/blueman-tpl-60.png',
+	'data/icons/pixmaps/blueman-tpl-70.png',
+	'data/icons/pixmaps/blueman-tpl-80.png',
+	'data/icons/pixmaps/blueman-tpl-90.png',
+	'data/icons/pixmaps/blueman-tpl-100.png',
+    install_dir: join_paths([datadir, 'pixmaps', package_name])
+)
+
+install_subdir(
+    'data/icons/hicolor',
+    install_dir: join_paths([datadir, 'icons'])
+)
+
+
+# GtkBuilder UI files
+install_subdir(
+    'data/ui',
+    install_dir: pkgdatadir
+)
+
+
+# Gsettings and pin database
+install_data('data/org.blueman.gschema.xml', install_dir: schemadir)
+install_data('data/pin-code-database.xml', install_dir: pkgdatadir)
+
+
+# Merge translations into desktop files
+i18n.merge_file(
+    input: 'data/blueman.desktop.in',
+    output: 'blueman.desktop',
+    type: 'desktop',
+    po_dir: 'po',
+    install: true,
+    install_dir: join_paths([get_option('sysconfdir'), 'xdg', 'autostart']),
+)
+
+i18n.merge_file(
+    input: 'data/blueman-adapters.desktop.in',
+    output: 'blueman-adapters.desktop',
+    type: 'desktop',
+    po_dir: 'po',
+    install: true,
+    install_dir: join_paths([datadir,'applications']),
+)
+
+i18n.merge_file(
+    input: 'data/blueman-manager.desktop.in',
+    output: 'blueman-manager.desktop',
+    type: 'desktop',
+    po_dir: 'po',
+    install: true,
+    install_dir: join_paths([datadir,'applications']),
+)
+
+i18n.merge_file(
+    input: 'data/thunar-sendto-blueman.desktop.in',
+    output: 'thunar-sendto-blueman.desktop.desktop',
```

Comment:
!

Reply:
Good catch, the old copy paste error :disappointed: